index.js

```diff
@@ -0,0 +1,311 @@
+    let newPriority = taskField.children[index].getElementsByTagName("small")[0];
+    let newTime = taskField.children[index].getElementsByTagName("small")[1];
+    newTitle.innerHTML = task.title;
+    newText.innerHTML = task.text;
+    newPriority.innerHTML = selectPriority() + ' priority';
+    newTime.innerHTML = setTaskTime();
+    colorTask(li, selectPriority() + ' priority');
+    editTaskButton.setAttribute("data-dismiss", "modal");
+    saveData();
+}
+
+function deleteTaskData() {
+    let currentTask = this.closest('.list-group-item');
+    index = Array.from(currentTask.parentNode.children).indexOf(currentTask);
+    let type = this.closest('.list-group-item').parentNode.id;
+    if (type === "currentTasks") {
+        toDo.splice(index, 1);
+        taskField.removeChild(currentTask);
+        toDoCount--
+        document.getElementById('toDo-count').innerHTML = ' (' + toDoCount + ')';
+    } else {
+        completed.splice(index, 1);
+        completeField.removeChild(currentTask);
+        completedToDoCount--
+        document.getElementById('completed-count').innerHTML = ' (' + completedToDoCount + ')';
+    }
+    saveData();
+}
+
+function sortFromNew() {
+    toDo.sort(function(a, b) {
+        return new Date(a.date) - new Date(b.date);
+    });
+    taskField.innerHTML = "";
+    toDo.forEach(function(el) {
+        return createTaskField(el, taskField);
+    })
+    saveData();
+}
+
+function sortFromOld() {
+    toDo.sort(function(a, b) {
+        return new Date(b.date) - new Date(a.date);
+    });
+    taskField.innerHTML = "";
+    toDo.forEach(function(el) {
+        return createTaskField(el, taskField);
+    })
+    saveData();
+}
+
+function changeTheme() {
+    if (switcher.checked) {
+        document.querySelector('.navbar').classList.remove('bg-light');
+        document.querySelector('.modal-content').style.backgroundColor = '#0e1621';
+        document.body.style.backgroundColor = '#0e1621';
+        document.body.style.color = 'white';
+        switcher.nextElementSibling.textContent = 'Dark Mode';
+    } else {
+        document.querySelector('.navbar').classList.add('bg-light');
+        document.querySelector('.modal-content').style.backgroundColor = 'white';
+        document.body.style.backgroundColor = 'white';
+        document.body.style.color = 'black';
+        switcher.nextElementSibling.textContent = 'White Mode';
+    }
+}
+
+
+newTask.addEventListener('click', createNewTask)
+addTaskButton.addEventListener('click', addTask);
+editTaskButton.addEventListener('click', saveEditChanges);
+newSort.addEventListener('click', sortFromNew);
+oldSort.addEventListener('click', sortFromOld);
+switcher.addEventListener('click', changeTheme)
+
+
+function saveData() {
+    localStorage.setItem('toDo', JSON.stringify(toDo));
+    localStorage.setItem('completedToDo', JSON.stringify(completed));
+}
+
+
+window.onload = function loadTasks() {
+    if (localStorage.getItem('toDo') !== null) {

```

Comment:
let task = localStorage.getItem('toDo');
if(task) { ...

toDo = JSON.parse(task)
...

Reply:
Thanks!
i fixed it!
```suggestion
let task = localStorage.getItem('toDo');
    if (task !== null) {
```